app/backend/middleware/graphql/index.ts

```diff
@@ -0,0 +1,58 @@
+import type { Request } from 'express';
+import { postgraphile } from 'postgraphile';
+import { pgPool } from '../../db/setup-pg';
+import { makePluginHook } from 'postgraphile';
+import PgManyToManyPlugin from '@graphile-contrib/pg-many-to-many';
+import PostgraphileLogConsola from 'postgraphile-log-consola';
+import ConnectionFilterPlugin from 'postgraphile-plugin-connection-filter';
+import { TagsFilePlugin } from 'postgraphile/plugins.js';
+import PostGraphileUploadFieldPlugin from 'postgraphile-plugin-upload-field';
+import PgOmitArchived from '@graphile-contrib/pg-omit-archived';
+import PgOrderByRelatedPlugin from '@graphile-contrib/pg-order-by-related';
+import authenticationPgSettings from './authenticationPgSettings';
+
+// Use consola for logging instead of default logger
+const pluginHook = makePluginHook([PostgraphileLogConsola]);
+
+let postgraphileOptions = {
+  pluginHook,
+  classicIds: true,
+  enableQueryBatching: true,
+  dynamicJson: true,
+  extendedErrors: ['hint', 'detail', 'errcode'],
+  showErrorStack: true, // setting this to "json" results in a typescript error
+  graphiql: true,
```

Comment:
Let's add a card to review if we need graphiql deployed in prod? For now, having graphiql in our dev environment is immensely helpful.